addons/mail/static/src/models/message/message.js

```diff
@@ -454,6 +409,25 @@ function factory(dependencies) {
             return parseAndTransform(prettyBody, addLink);
         }
 
+        /**
+         * @private
+         * @returns {mail.thread[]}
+         */
+        _computeThreads() {
+            const cmd = [['replace', []]];
+            cmd.push(['link', this._threads]);
+            if (this.originThread) {
+                cmd.push(['link', this.originThread]);
+            }
+            cmd.push([this.isHistory ? 'link' : 'unlink', this.env.messaging.history]);
+            cmd.push([this.isNeedaction ? 'link' : 'unlink', this.env.messaging.inbox]);
+            cmd.push([this.isStarred ? 'link' : 'unlink', this.env.messaging.starred]);
+            if (this.env.messaging.moderation) {
```

Comment:
I think this key is pointless past the fact that the mail box moderation is created or not at init.

Reply:
I don't get what you mean by "this key".

Reply:
I meant "this attr" but comment was incorrect, ignore it.